COLLABORATOR_GUIDE.md

```diff
@@ -200,6 +225,20 @@ changing error messages in any way, altering expected timing of an event (e.g.
 moving from sync to async responses or vice versa), and changing the
 non-internal side effects of using a particular API.
 
+Purely additive changes (e.g. adding new events to EventEmitter
+implementations, adding new arguments to a method in a way that allows
+existing code to continue working without modification, or adding new
+properties to an options argument) are handled as semver-minor changes.
+
+Note that errors thrown, along with behaviors and APIs implemented by
+dependencies of Node.js (e.g. those originating from V8) are generally not
+under the control of Node.js and therefore *are not directly subject to this
+policy*. However, care should still be taken when landing updates to
```

Comment:
They're not subject to the deprecation policy, but they are still tagged `semver-major`, still need the TSC signoffs, and still don't get backported to any release lines right? Why the move up to this section?

As a side note, it seems odd that this section doesn't mention that you have to put the `semver-major` label on the PR (I know it wasn't there before). Maybe it's covered somewhere else.

Reply:
Hmm, yes this seem to belong to the "Breaking Changes and Deprecations" section. Thanks for catching that.